packages/liquidator/test/Liquidator.js

```diff
@@ -28,693 +29,740 @@ const Token = artifacts.require("ExpandedERC20");
 const Timer = artifacts.require("Timer");
 
 contract("Liquidator.js", function(accounts) {
-  // Implementation uses the 0th address by default as the bot runs using the default truffle wallet accounts[0].
-  const liquidatorBot = accounts[0];
-  const sponsor1 = accounts[1];
-  const sponsor2 = accounts[2];
-  const sponsor3 = accounts[3];
-  const contractCreator = accounts[4];
-
-  let collateralToken;
-  let emp;
-  let liquidator;
-  let syntheticToken;
-  let mockOracle;
-  let priceFeedMock;
-
-  let spy;
-  let spyLogger;
-
-  let liquidatorConfig;
-  let liquidatorOverridePrice;
-  let empProps;
-
-  before(async function() {
-    collateralToken = await Token.new("UMA", "UMA", 18, { from: contractCreator });
-    await collateralToken.addMember(1, contractCreator, {
-      from: contractCreator
+  describe("18 decimal collateral", function() {
```

Comment:
Since it seems that git isn't processing this diff super well (my guess is that it's because of the indentation or something). Do you think you could add one or two PR comments directing us to what blocks of the diff can be ignored and which parts are "new" and we should be looking at?

Reply:
Oh oops, @nicholaspai asked for the same thing lol

Reply:
it's literally due to this `describe` block that the diff is so big. This indentes all code within it from 2 to 4 spaces.

Reply:
in github you can set this to hid the white space changes:

![image](https://user-images.githubusercontent.com/12886084/91966339-fd698780-ed11-11ea-8db1-a672fa8a82de.png)